build.gradle

```diff
@@ -32,6 +32,7 @@ dependencies {
     
     testImplementation('org.springframework.boot:spring-boot-starter-test')
     testImplementation('com.h2database:h2')
+    testImplementation group: 'org.hamcrest', name: 'java-hamcrest', version: '2.0.0.0'
```

Comment:
sposób zapisu nie spójny z tym linię wyżej

Reply:
Ja to skopiowałem poprostu z MavenRepo ale ok zmieniłem na taki sposób zapisu jak wyżej...